components/home/home-template.html

```diff
@@ -1,27 +1,27 @@
- <div class="row">
-  <H2>HOME</H2><BR>
+<div class="row">
+  <H2>HOME</H2><BR/>
 </div>
 
 <div class="row" align="center">
-  <div contenteditable>
-   <h1><span style="color: orange">R</span>ussell Gilbert <span style="color: orange">L</span>. <span style="color: orange">D</span>e Castro</h1>
-  </div>
-  <p>Programmer</p>
-  <br>
+    <div contenteditable>
+     <h1><span style="color: orange">R</span>ussell Gilbert <span style="color: orange">L</span>. <span style="color: orange">D</span>e Castro</h1>
```

Comment:
This is looking great so far.  Let's update and remove all of your inline css `style=` and start using CSS sheets for each template.  Might as well use LESSJS or SASS now also.  So for example, `home-template.html` should have an accompanying `home-template.scss` or `home-template.less` depending on which you choose.  They are both very similar.  I use SASS (http://sass-lang.com/).